store/android/src/main/sqldelight/com/jakewharton/sdksearch/store/Item.sq

```diff
@@ -2,20 +2,24 @@ import java.lang.Boolean;
 
 CREATE TABLE item(
   id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
-  listing TEXT NOT NULL,
   packageName TEXT NOT NULL,
   className TEXT NOT NULL,
   deprecated INTEGER AS Boolean NOT NULL DEFAULT 0,
-  link TEXT NOT NULL
+  link TEXT NOT NULL,
+
+  CONSTRAINT fqcn UNIQUE (packageName, className)
```

Comment:
you dont need to name the constraint just `UNIQUE (packageName, className)` should work fine